packages/11ty/_plugins/figures/schema.json

```
@@ -0,0 +1,175 @@
+{
+  "$id": "https://quire.getty.com/figures.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$ref": "#/definitions/Figures",
+  "title": "Figures",
+  "description": "Quire figures data schema",
+  "type": "object",
+  "definitions": {
+    "Figures": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "figure_list": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FigureList"
```

```suggestion
            "$ref": "#/definitions/Figure"
```
Might be more clear if this object definition is `Figure` since it's the item not the list?